metadata/metadata-modeshape/src/main/java/com/thinkbiganalytics/metadata/modeshape/BaseJcrProvider.java

```diff
@@ -355,12 +408,79 @@ public void deleteById(PK id) {
         };
     }
 
+    protected StringBuilder startBaseQuery(String... props) {
+        String alias = getEntityAlias();
+        StringBuilder bldr = new StringBuilder("SELECT ");
+        if (props.length == 0) {
+            bldr.append(alias).append(".* ");
+        } else {
+            boolean start = true;
+            for (String prop : props) {
+                if (! start) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this condition so that it does not always evaluate to "false" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2583)